src/components/LiveRates/LiveRates.scss

```diff
@@ -0,0 +1,67 @@
+@use "../../assets/sass/_variables.scss" as *;
+
+.live-rates {
+  margin-left: 5%;
+
+  &__h4 {
+    color: $color-middle-blue;
+    font-family: $title-font-family;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 15px;
+    margin-bottom: 0px;
+  
+  }
+
+  &__h2{
+    font-family: $title-font-family;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 32px;
+    line-height: 40px;
+  }
+  &__para {
+    font-family: $third-font-family;
+    width: 514px;
+    height: 72px;
+    font-style: normal;
+    font-size: 16px;
+    line-height: 150%;
+    margin-top: 0px;
+    margin-bottom: 30px;
+    color: $color-font-lightgray;
+  }
+}
+
+
+
+.button {
```

Comment:
button styling section - not nested under live rates?

Reply:
This particular button "Add Currency" does not need to be nested because it is different from the buttons on the actual LiveRatesList table.